src/signals/settings/users/Detail/components/UserForm/__tests__/UserForm.test.js

```diff
@@ -75,6 +78,8 @@ describe('signals/settings/users/containers/Detail/components/UserForm', () => {
     expect(container.querySelector('[name="is_active"][value="true"]').checked).toBe(true);
     expect(container.querySelector('[name="is_active"][value="false"]').checked).toBe(false);
     expect(container.querySelector('[name="note"]').value).toBe(data.profile.note);
+    expect(container.querySelectorAll('[name="departments"]')[0].checked).toBe(true);
+    expect(container.querySelectorAll('[name="departments"]')[7].checked).toBe(false);
```

Comment:
Can the index be inferred from the values in `departmentsJson`?

Reply:
Done